### PR TITLE
registering Swarnima Shukla

### PR DESCRIPTION
**Pull request by Swarnimashukla**

### CODE REVIEW

File: content/participants/ttjc@30214391/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Swarnima Shukla
+type: "participant"
+name: "Swarnima Shukla"
+portfolio: "https://swarashukla65.myportfolio.com/"
+github-link: "https://github.com/Swarnimashukla"
+linkedin-link: "https://www.linkedin.com/in/swarnima-shukla-3815b5b8/"
+looking-for: "job of my interest"
```
Comment (khusharth) [inline content/participants/ttjc@30214391/index.md:8]: Hey! @Swarnimashukla please use only **job** here.

File: content/participants/ttjc@30214391/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+I love dancing and sometimes reading novels (developing this habbit). I like to take challenges and sometimes don't think much about how compatible I am because I know I will sail through everything. I am nervous back-stage and  confident when it comes to final peformance .
+
+### help I need?
+
+It would be good if someone tells me how to search for jobs properly as I find most of jobs asking for work experience. Also I need a mock job interview focused on web development.
+
+### how to contact me?
+
+- [Twitter] (https://twitter.com/swarnimashukla5)
```
Comment (khusharth) [inline content/participants/ttjc@30214391/index.md:33]: Please use correct syntax for adding a link in markdown. There should be no space between [ ] and () brackets.

**Comment by Swarnimashukla** [comment]
@khusharth  I have made necessary changes please review it

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Comment by gkanishk** [comment]
Welcome, On-board!!!!

**Pull request merged**